### ISSUE TITLE
Upgrade nodejs in OpenSDN Web UI

### ISSUE THREAD

**Issue by aignatov** [OpenSDN-io/community#32]
The latest version of nodejs used in WEB UI is pretty much outdated: https://gerrit.opensdn.io/plugins/gitiles/opensdn-io/tf-packages/+/refs/heads/master/rpm/contrail-web-core/contrail-web-core.spec#34

This feature request aims to bump nodejs to the latest supported version:
https://nodejs.org/en/about/previous-releases

**Comment by randybias** [comment]
Have you made any attempts to see if the upgrade will work out of the box?  Are there possible downstream effects or issues?

**Comment by aignatov** [comment]
@randybias No, it doesn't work out of the box. We were able to upgrade nodejs WebUI to v16.x, it requires some code changes as well as rebuild of third-party-packages.

Btw nodejs v16.x is EOL and further upgrade will require new base images. It's because centos7 doesn't compile some libs for new nodejs.

**Comment by randybias** [comment]
Can we push that code back?
